src/zkevm_specs/evm/table.py

```diff
@@ -18,6 +18,7 @@ class FixedTableTag(IntEnum):
     table.
     """
 
+    Range5 = auto()  # value, 0, 0
```

Comment:
Why do we need a RangeCheck of [0,4]?? I'm not seeing it.

Reply:
putting all logs (log0--log4) codes into one `log` code, to this range check to ensure `log` code within range.

Reply:
Ohhh I see.
Hmmm I guess in the Rust impl we will probably use const-generics. But not sure that it will mean any change for that case.

So far looks correct to have this check here within this implementation.